src/Microsoft.AspNetCore.Razor.LanguageServer/BackgroundDocumentGenerator.cs

```diff
@@ -179,9 +180,6 @@ private async void Timer_Tick(object state)
             {
                 _foregroundDispatcher.AssertBackgroundThread();
 
-                // Timer is stopped.
-                _timer.Change(Timeout.Infinite, Timeout.Infinite);
```

Comment:
If this null-refing line is entirely gone, what is the point of the extra logging in the catch? Or was this a necessary change?